doc/manual/manual.tex

```diff
@@ -7073,7 +7073,68 @@ \subsubsection{Melt migration in a 2D mid-ocean ridge model}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.9\textwidth]{cookbooks/kinematically_driven_subduction_2d_case1/doc/setup_Quinquis2014.png}
+    \caption{Case 4 model setup. Copied from Quinquis (2014).}
+    \label{fig:QQ_setup}
+\end{figure}
+
+\paragraph{Case 1 - Simple geometry and rheology}
+The Case 1 model setup considers 3 materials (compositional fields) apart from the background sublithospheric mantle (see Fig.~\ref{fig:QQ_case1_setup}): 
+\begin{enumerate}
+    \item the lithosphere of the overriding plate (combining the BOC, SHB and thermal layer of the overriding plate in Fig.~\ref{fig:QQ_setup}),
+    \item the crust of the subducting plate (weak seed and BOC combined), and
+    \item the mantle lithosphere of the subducting plate (SHB and thermal layer combined).
+\end{enumerate}{}
+The geometry of these compositions is implemented as follows:
+\lstinputlisting[language=prmfile]{cookbooks/kinematically_driven_subduction_2d_case1/Case1_compositions.prm}
+
+No differences in material properties exist, except for density and viscosity, so we use the multicomponent material model. To keep the boundaries between fields as sharp as possible in terms of viscosity, we use the maximum composition to determine the viscosity in each evaluation point (note that this can be harder on the solver):
+\lstinputlisting[language=prmfile]{cookbooks/kinematically_driven_subduction_2d_case1/Case1_materialmodel.prm}
+
+Temperature effects are ignored. Subduction is driven by prescribed in- and outflow through the right boundary (with a gradual transition of the flow direction), all other boundaries are free slip. The volume of material that flows in is balanced by the volume that is prescribed to flow out (this is important as the model is incompressible). A weak crust along the plate interface and the subducting lithosphere facilitates subduction. Through the function plugin, we prescribe the in- and outflow:
+\lstinputlisting[language=prmfile]{cookbooks/kinematically_driven_subduction_2d_case1/Case1_velocity.prm}
+
+To follow the slab on its descent into the mantle, we use adaptive mesh refinement based on viscosity i.c.w. the minimum refinement strategy to ensure sufficient resolution in the crust at the surface and the weak zone that allow the slab to detach from the surface:
```

Comment:
what does "i.c.w." mean? In conjunction with? This is not a common abbreviation in American English, I believe.

Reply:
```suggestion
To follow the slab on its descent into the mantle, we use adaptive mesh refinement based on viscosity i.c.w. the minimum refinement strategy to ensure sufficient resolution in the crust at the surface and the weak zone that allows the slab to detach from the surface:
```